eduvpn/actions/delete.py

```diff
@@ -5,18 +5,18 @@
 
 import logging
 import gi
-gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk, GLib
 from eduvpn.manager import delete_provider
 from eduvpn.notify import notify, init_notify
 from eduvpn.util import error_helper, metadata_of_selected
 from eduvpn.steps.start import refresh_start
 
 
+gi.require_version('Gtk', '3.0')
```

Comment:
dit statement moet voor de import van Gtk. Dat gaat nu misschien goed omdat er een andere module het versie nummer misschien al heeft geset.

Dit soort veranderingen lijken misschien triviaal, maar het is belangrijk om een PR single purpose te houden.